docker-images/docker-compose.yml

```diff
@@ -12,8 +12,17 @@
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
 
-version: '3.8'
+version: '3.7'
```

Comment:
Ran into this issue on AL2 as well, longer term concerns on having to downgrade.

Reply:
For me I think it was because my local version of docker-compose didn't support 3.8 yet. It should be ok as long as we don't need features available only in later schemas.